### PR TITLE
deprecate CAS in favor of CompareAndSwap

### PR DESCRIPTION
**Pull request by icarus-ai**
deprecate CAS in favor of CompareAndSwap, since that's the convention
the standard library chose for these in Go 1.19

### CODE REVIEW

**Review by Mrs4s — APPROVED** [review]

**Comment by icarus-ai** [comment]
忘了改workflows里go的版本😓

**Comment by Mrs4s** [comment]
go 1.19 竟然都发布了

File: client/statistics.go [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package client
 
-import (
-	"go.uber.org/atomic"
-)
+import "sync/atomic"
 
 type Statistics struct {
 	PacketReceived  atomic.Uint64 `json:"packet_received"`
```
Comment (wdvxdr1123) [inline client/statistics.go:6]: sync/atomic.Uint64 不支持序列化， 请为 Statistics 实现 MarshalJSON 接口

**Comment by wdvxdr1123** [comment]
See 2d79fccd1060158a60ffc76058c01860e3b83465

**Pull request merged**